tools/apiview/parsers/CONTRIBUTING.md

```diff
@@ -1,1165 +1,44 @@
+A sample token file is present [here](./apiview-treestyle-parser-schema/sample/Azure.Template_token.json).
 
-- Each module of the API (namespace, class, methods) should be its own node. Members of a module (methods in a class), (classes in a namespace) should be added as child nodes of its parent module.
-- Each tree node has top tokens which should be used to capture the main tokens on the node, these can span multiple lines.
-- Module name, decorators, and parameters should be modeled as `TopTokens`.
-- If the language requires it use the bottom tokens to capture tokens that closes out the node, this is usually just the closing bracket and/or empty lines.
+APIView generates navigation tree based on the information in token. A token is included in the navigation tree if `NavigatonDisplayName` is set in `ReviewToken` and `LineId` is set in `ReviewLine` object that contains the `ReviewToken`
 
-### Object Definitions
-
-- Here are the models needed
-
-  ```
-  object APITreeNode
-    string Name (Required)
-    string Id (Required)
-    string Kind (Required)
-    Set<string> Tags
-    Dictionary<string, string> Properties
-    List<StructuredToken> TopTokens
-    List<StructuredToken> BottomTokens
-    List<APITreeNode> Children
-
-  object StructuredToken
-    string Value
-    string Id
-    StructuredTokenKind Kind (Required)
-    Set<string> Tags
-    Dictionary<string, string> Properties 
-    Set<string> RenderClasses 
-
-  enum StructuredTokenKind
-    Content
-    LineBreak
-    NonBreakingSpace
-    TabSpace
-    ParameterSeparator
-  ```
-
-### [APITreeNode](../../../src/dotnet/APIView/APIView/Model/TokenTreeModel.cs)
-- Ensure each node has an Id and Kind. The combination of `Id`, `Kind`, and `SubKind` should make the node unique across all nodes in the tree. For example a class and a method can potentially have the same Id, but the kind should differentiate them from each other.
-- Sort each node at each level of the tree by your desired property, this is to ensure that difference in node order does not result in diff.
-
-### [StructuredToken](../../../src/dotnet/APIView/APIView/Model/StructuredTokenModel.cs)
-
-- Assign the final parsed value to a `List<APITreeNode> APIForest` property of the `CodeFile`
 
 ## Serialization
 
-Serialize the generated code file to JSON with Gzip compression. The output file should have `.json.tgz` extension. Try to make the json as small as possible by ignoring null values and empty collections.
-Don't worry about indentation that will be handled by the tree structure. In the case you want to have indentation between the tokens then use `TabSpace` token kind.
+Serialize the generated code file to JSON. The output file should have `.json` extension. Try to make the json as small as possible by ignoring null values and empty collections.
+Don't worry about indentation that will be handled by the tree structure based on the parent - child relationship among `ReviewLine` objects.
```

Comment:
```suggestion
Don't worry about indentation as it will be handled by the tree structure based on the parent-child relationship among `ReviewLine` objects.
```